src/utils/sendEmail.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * Takes an email and sends a purchase receipt to that email, including
+ * each item's name, quantity, and price (individual price * quantity), date
+ * of purchase, and order/purchase number
+ * 
+ * Parameters:
+ *    String email: email to send receipt to
+ *    Array item: array of arrays: [name, qty, price]
+ *                (best if price has $ sign) (price is total price of that item)
+ *    String date: date
+ *    String orderNum: order number
+ */
+const fs = require('fs');
+const jsdom = require('jsdom');
+const { JSDOM } = jsdom;
+
+async function sendEmail(email, items, date, orderNum) {
+   return new Promise((resolve, reject) => {
+      var emailSent = false;
+      var emailContent = fs.readFileSync('./src/utils/emailReceipt.html', 'utf8');
+
+      const populatedEmail = populate(emailContent, items, date, orderNum);
+
+      // Send email using sendgrid
+      const sgMail = require('@sendgrid/mail')
+      sgMail.setApiKey(process.env.SENDGRID_API_KEY)
+      const msg = {
+         to: email,
+         from: 'ryan_yychen@hotmail.com',
+         subject: 'HKN Snack Bar Receipt',
+         text: 'nothing for now',
+         html: populatedEmail,
+      }
+      sgMail
+         .send(msg)
+         .then(() => {
+            emailSent = true;
+            if (emailSent) { // if email sent successfully
+               resolve(true);
+            } else { // email failed to send
+               reject(new Error("Email failed to send"));
+            }
+         })
+         .catch((error) => {
+            console.error(error);
```

Comment:
Change this to `reject(error)` since this would get returned out of the function and displayed in a `catch()` block to the caller.